landoapi/api/treestatus.py

```diff
@@ -110,20 +104,18 @@ def wrap_output(*args, **kwargs) -> tuple[dict[str, Any], int]:
     return wrap_output
 
 
-def serialize_last_state(old_tree: dict, new_tree: CombinedTree) -> str:
+def serialize_last_state(old_tree: dict, new_tree: CombinedTree) -> dict[str, Any]:
     """Serialize a `last_state` value for a `StatusChangeTree`."""
```

Comment:
nit: this docstring is missing some context, and doesn't impart any information other than what is in the name of this method. Perhaps add some additional documentation.